server/cluster/cluster.go

```diff
@@ -295,11 +297,81 @@ func (c *RaftCluster) Start(s Server) error {
 	go c.runMinResolvedTSJob()
 	go c.runSyncConfig()
 	go c.runUpdateStoreStats()
+	go c.startGCTuner()
 
 	c.running.Store(true)
 	return nil
 }
 
+// startGCTuner
+func (c *RaftCluster) startGCTuner() {
+	defer logutil.LogPanic()
+	defer c.wg.Done()
+
+	tick := time.NewTicker(10 * time.Second)
+	defer tick.Stop()
+	totalMem, err := memory.MemTotal()
+	if err != nil {
+		log.Fatal("fail to get MemTotal:%s", zap.Error(err))
```

Comment:
```suggestion
		log.Fatal("fail to get MemTotal:%s", zap.Error(err))
```
How about using the uniform format for the log?